clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -33,8 +31,10 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.function.BiConsumer;
+import java.util.function.Function;
```

Comment:
Sorry for the un-related changes :( happy to revert if it adds too much noise during the review.